feat: adds initial implementation for the fedramp-transform command

jpower432 opened the pull request:
## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation. (`compliance-trestle` PR)
- [X] I have updated the documentation accordingly. . (`compliance-trestle` PR)
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.
- [X] All commits are signed-off.

## Summary

Blocked by #30 

Partially addresses #20 

This creates the `fedramp-transform` command and the initial classes.
This only populates the `control origination` values.

The description, implementation status, and parameter values will be on follow-on PRs.

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

vikas-agarwal76 commented on trestle_fedramp/core/ssp_reader.py:
```diff
@@ -0,0 +1,193 @@
+    CUST_CONFIGURED = 'customer-configured'
+    CUST_PROVIDED = 'customer-provided'
+    INHERITED = 'inherited'
+
```
Should these short names be in a const file? Also are these defined by FedRAMP or us?

jpower432 replied:
These are defined by FedRAMP [here](https://github.com/GSA/fedramp-automation/blob/45479230b1fa712b5a8443a634593fcef1a14e44/src/content/rev5/resources/xml/fedramp_values.xml). I went back and forth over whether I we should define them here or parse the xml file. I ended up defining them since the values seem to be very stable, but I can also see how that would not scale as easily as we add more FedRAMP specific values. Interested in your thoughts since `fedramp` resources are already being copied over.

jpower432 replied:
Also wanted to add that the control origination values in the xml do not cover the `hybrid` or `shared` values that are check boxes on the template, but the FedRAMP SSP template shows examples with multiple control origination values. So the logic on handling multiple control origination values and how they translate to `hybrid` and `shared` is defined here.

jpower432 commented:
@vikas-agarwal76 I addressed your comment about putting the short-labels in the `const.py`. I also added support for multiple boxes to be checked at a time to better align with real word use cases.

jpower432 commented:
Closing in favor of #39
